src/components/TagList/TagList.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from 'react';
+
+import classNames from 'classnames';
+
+import { Icon } from '../Icon/Icon';
+
+export interface TagListProps {
+	tags: string[];
+	swatches?: boolean;
+	bordered?: boolean;
+	closable?: boolean;
+	onTagClosed?: (tag: string) => void;
+}
+
+export const TagList: FunctionComponent<TagListProps> = ({
+	tags,
+	swatches = true,
+	bordered = true,
+	closable = false,
+	onTagClosed = () => {},
```

Comment:
maken we het niet de verantwoordelijkheid van de tag zelf om zichzelf te wissen indien de sluit knop is ingedrukt, ipv enkel een event te smijten. Of is er een usecase waar dit niet gewenst is?

Reply:
We moeten dit event sowieso behouden voor side-effects vanuit de parent. Ik zou de logica daarvoor ook bij de parent steken en niet in de TagList zelf, aangezien de doorgegeven props dan altijd overeen gaan komen met hetgeen gerenderd wordt.